src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -108,6 +112,15 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 	get activeCellId(): string {
 		return this._activeCellId;
 	}
+	/**
+	 * Returns the code editor of makrdown cell in edit mode.
+	 */
+	getEditor(): BaseTextEditor | undefined {
+		if (this.markdowncodeCell.length > 0) {
+			return this.markdowncodeCell.toArray()[0].getEditor();
```

Comment:
You should be able to just use the .first property instead of converting the whole thing to an array.